CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to contribute
+
+If you would like to make a contribution to protractor-sync, please check the following steps and guidelines
+ 
+## Installation
+
+Clone the repo
+
+`git clone https://github.com/blackboard/protractor-sync.git`
+
+## Making Changes
+
+Create a branch that summarizes the work you are submitting
+
+`git checkout -b feature/some-protractor-sync-branch`
+
+When adding new features, be sure to add a test to  `protractor-sync/test/spec/protractor_sync_test.ts`
+
+Summarize changes in commit message 
+
+## Submitting Changes 
+
+Push changes 
+
+`git push origin feature/some-protractor-sync-branch`
```

Comment:
I'm pretty sure this won't work for someone who is not part of the Bb organization. Since they won't have write permissions to the repo, they won't be able to push a new branch. Instead, they will want to fork the project and create a PR from their fork to our repo. This also means that while the 'Clone the repo' step above might work for them (I'm not sure), it's probably not what they want to do. They'll want to pull down their own fork instead of cloning our repo.

Something like https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/ or https://gist.github.com/MarcDiethelm/7303312 does a nice job of explaining the whole process.